dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/serial/SerializingExecutor.java

```diff
@@ -86,6 +86,9 @@ private void schedule(Runnable removable) {
                         // to execute don't succeed and accidentally run a previous runnable.
                         runQueue.remove(removable);
                     }
+                    if (removable instanceof CloseableRunnable) {
```

Comment:
It will not be executed when `atomicBoolean.compareAndSet(false, true) == false (line 68)`